docs/examples/ex18_square.py

```diff
@@ -0,0 +1,84 @@
+from skfem import *
```

Comment:
Should this file replace ex18.py or be separate?

Reply:
Add to the test suite?

Reply:
Sorry, ex18_square was just a preliminary sketch which I thought I had deleted.